solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -114,4 +114,62 @@ print(ans)
 ```
 
 </PySection>
+
+<JavaSection>
+```java
```

Comment:
looks like formatting got a bit messed up here, make sure you're using tabs, not spaces for indentation

Reply:
yah..... I'll try to fix it. For some reason, when I push from vsCode or copy directly from the guide editor, the styling gets wonky. Also, it's weird because github tabs are huge and don't indent correctly

Reply:
yeah one way u can resolve this is by setting indentation on vscode to tabs so you wont get any issues when using vscode

Reply:
Thank you!